ansible/roles/images/tasks/load.yaml

```diff
@@ -0,0 +1,5 @@
+
+- name: import images from local cache
+  shell: "ctr -n k8s.io image import {{ item }}"
```

Comment:
Can it be `cmd`?
```suggestion
  cmd: "ctr -n k8s.io image import {{ item }}"
```

Reply:
Hmm. I see we're using this form everywhere else:

```yaml
shell:
  cmd:  "..."
```

I'll change to that form.

Or are you asking about using command module, instead of the shell module?

Reply:
(Changed from the shell to the command module, unless I needed redirection)